src/Kdyby/Redis/Driver/PhpRedisSharedDriver.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Redis\Driver;
+
+use Kdyby;
+use Nette;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class PhpRedisSharedDriver extends PhpRedisDriver
+{
+
+	/**
+	 * @var integer
+	 */
+	private $database = 0;
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function select($database)
```

Comment:
Actually, this could be even in the `PhpRedisDriver`. Having the driver remember last used database is a nice feature.